### PR TITLE
Feat/rememberrecentproject

### PR DESCRIPTION
**Pull request by ilyadusoleil**

### CODE REVIEW

File: server/config/database.js [hunk 1]
```diff
@@ -6,7 +6,7 @@ module.exports = {
   development: {
     username: process.env.DB_USER,
     password: process.env.DB_PASS,
-    database: 'zapp_development',
+    database: 'iazrvlap',
```
Comment (SteHeRa) [inline server/config/database.js:9]: ?

**Review by SteHeRa — COMMENTED** [review]
👍

File: server/config/database.js (hunk 1 above)
Comment (ilyadusoleil) [reply server/config/database.js:9]: oh whoops i'll revert this

File: server/config/database.js [hunk 2]
```diff
@@ -6,7 +6,7 @@ module.exports = {
   development: {
     username: process.env.DB_USER,
     password: process.env.DB_PASS,
-    database: 'iazrvlap',
+    database: process.env.DB_NAME_DEV,
```
Comment (SteHeRa) [inline server/config/database.js:9]: 👍

**Review by SteHeRa — COMMENTED** [review]
👌✔

**Pull request merged**